messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/DatabusConsumer.java

```diff
@@ -131,10 +133,37 @@ protected void initializeConfig(ClientConfig config) throws IOException {
         + " streamType:" + streamType);
   }
 
+  private void startNewCollectorDiscovererTimer(ClientConfig config) {
+       /*
+    This timer is responsible for identifying new
+    collector output files and initialising partition readers for them
+     */
+    int initialDelay = config.getInteger(initialDelayForDiscoverer,
```

Comment:
@sreedishps Again, why two configs are being asked for periodic discovery? We should minimize the number of configs exposed because of internal implementation details (in this case Timer needs 2 inputs). 

As a user of messaging consumer, I would just want to provide one configurable frequency for this functionality.